make_iterator_from_pointer.hpp

```diff
@@ -0,0 +1,256 @@
+        return *this = wrap_pointer_to_const_iterator<T*>::operator-=(index);
+    }
+
+    template<typename T>
+    wrap_pointer_to_iterator<T*> wrap_pointer_to_iterator<T*>::operator-(difference_type index) const {
+        return wrap_pointer_to_const_iterator<T*>::operator-(0);
+    }
+
+
+    template<typename T>
+    typename wrap_pointer_to_const_iterator<T*>::reference wrap_pointer_to_const_iterator<T*>::operator*() const {
+        return *_ptr;
+    }
+
+    template<typename T>
+    typename wrap_pointer_to_const_iterator<T*>::pointer wrap_pointer_to_const_iterator<T*>::operator->() const {
+        return addressof(operator*());
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> &wrap_pointer_to_const_iterator<T*>::operator++() {
+        ++_ptr;
+        return *this;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> wrap_pointer_to_const_iterator<T*>::operator++(int) {
+        wrap_pointer_to_const_iterator<T*> copy = *this;
+        ++(*this);
+        return copy;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> &wrap_pointer_to_const_iterator<T*>::operator--() {
+        --_ptr;
+        return *this;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> wrap_pointer_to_const_iterator<T*>::operator--(int) {
+        wrap_pointer_to_const_iterator<T*> copy = *this;
+        --(*this);
+        return copy;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> &wrap_pointer_to_const_iterator<T*>::operator+=(difference_type index) {
+        _ptr += index;
+        return *this;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> &wrap_pointer_to_const_iterator<T*>::operator-=(difference_type index) {
+        _ptr -= index;
+        return *this;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> wrap_pointer_to_const_iterator<T*>::operator+(difference_type index) const {
+        wrap_pointer_to_const_iterator copy = *this;
+        ++(*this);
+        return copy;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> wrap_pointer_to_const_iterator<T*>::operator-(difference_type index) const {
+        wrap_pointer_to_const_iterator copy = *this;
+        --(*this);
+        return copy;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*> wrap_pointer_to_const_iterator<T*>::operator[](difference_type index) const {
+        return *this + index;
+    }
+
+    template<typename T>
+    wrap_pointer_to_const_iterator<T*>::wrap_pointer_to_const_iterator(pointer ptr) {
+        _ptr = ptr;
+    }
+
+    template<typename T>
+    bool wrap_pointer_to_const_iterator<T*>::operator==(const wrap_pointer_to_const_iterator &other) const {
+        return _ptr == other._ptr;
+    }
+}
+
+#endif //LGHAZAROSYAN_MAKE_ITERATOR_FROM_POINTER_HPP
```

Comment:
always left one blank line at the en of file

Reply:
😒😒😒😒😒😑😑🙄🙄🙄🙄